crates/conjure_core/src/rules/minion.rs

```diff
@@ -405,6 +405,36 @@ fn introduce_modeq(expr: &Expr, _: &Model) -> ApplicationResult {
     )))
 }
 
+#[register_rule(("Minion", 4400))]
+fn introduce_abseq(expr: &Expr, _: &Model) -> ApplicationResult {
+    let (x, abs_y): (Atom, Expr) = match expr.clone() {
+        Expr::Eq(_, a, b) => {
+            let a_atom: Option<&Atom> = (&*a).try_into().ok();
```

Comment:
really looking forward to [box_patterns](https://github.com/rust-lang/rust/issues/29641) to avoid this kind of thing

Reply:
oh, apparently box_patterns is no more, it's going to be [deref patterns](https://github.com/rust-lang/rust/issues/87121) instead.